Ginger/GingerCoreNET/UserCofig/UserProfile.cs

```diff
@@ -468,8 +468,10 @@ public void SaveRecentAppAgentsMapping()
 
         public void LoadRecentAppAgentMapping()
         {
+            Console.WriteLine("Solution:" +WorkSpace.Instance.Solution.Name);
             if (WorkSpace.Instance.Solution != null)
             {
+                Console.WriteLine("Agent Mapping");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)